### ISSUE TITLE
Working 16 mm film camera 'weapons'

### ISSUE THREAD

**Issue by Original-OEM** [Savage-Game-Design/Mike-Force#174]
**Is your feature request related to a problem? Please describe.**
It is possible you're already working on this as the 1.00.3 addition of a 35mm stills camera maybe hints at. BTW it would be great either enable general saves of the 35mm camera shots, or if they are saved already then explain how users access them (excluding specific missions that use the camera). 

**Describe the solution you'd like**
Create a working in-game 'weapon' model of a 16mm film camera to record short clips using the same magazine function of a weapon, i.e. limited to say 20 mins per 'reel aka magazine/ammo' that the player has.
This would be similar to the modern VHS camera journo NPC models seen in vanilla ARMA3 but actually playable and in historical context much more of an actual weapon than a daisy cutter!
 
**Describe alternatives you've considered**
Include options with the camera that mimic weapons options:
1. Have 3 x mechanical zoom lens options to mirror typical 16 mm era cameras used in the field like Bolex etc. These would switch to 3 fixed zoom lens lengths using the same function as switching a weapons fire mode.
2. As mentioned, link recording length to ammo capacity - this would mimic era typical cameras and also prevent users filling their storage with huge videos. There could also be an option to include b/w and colour film reels maybe?
3. Again using the existing weapon model functions, include handheld as well as tripod mounts for the 16 mm film camera. Each providing different camera movement options, and also bulk for the player load out.
4. Optional to include a 'sound guy' with a boom mic to record spot sound if players want to team up as this role. This boom mic could include options such as be extendable and recording time to a tape recorder would be limited. There would also be the option to record sound only in-game which could enhance the role of 'sound-guy'.
5. Why not also add a '8 mm' amateur handheld camera option for grunts to use in field as some celebrated soldiers did officially and unofficially.
6. My thoughts are players competent in their own video editing would use the in-game recorded files in their own post-studios to make the final films. But the option to include an in-field developing and editing function could further enhance gameplay - but maybe this is for a future iteration?

**Additional context**
The Vietnam war was arguably the first conflict where news, stills and film footage had a significant impact on it's outcome. Everyone will have seen some imagery from the war and many will have seen or read stories of the camera crews, photographers, and journalists who recorded them. Famous fictional examples like the Apocalypse Now! appearance of the actual director posing as a journalist with his crew, and the emotional post AO battle recording of Hamburger Hill demonstrate some of the game-play options. Having a working camera and film camera model in-game would salute the actual people who risked their lives in many cases to show the world what was happening, and also echo the messages from ARMA3 laws of war in some respects.
Making these models playable would add to the game play with presumably minimal coding to make it fit the existing weapons model code. The essence of Mike Force when it works well is team playing and so being able to record that team play and share it on Discords or to the wider gaming public - but in an authentic (and easier!) way of actually being there with the other players as opposed to just screenshots with 4th wall breaking in-game menu text, or 3rd person and impossible view screen recordings - player made films like these would celebrate, share, and promote the Mike Force experience to even more ARMA and other players.

In conclusion you may already be working on this - if so then I'd love to beta test it for you - but if not then me and a few players I chatted to about this would ditch their M16's and grab a camera for their next AO in a heartbeat!
